src/phoenix/config.py

```diff
@@ -76,3 +77,7 @@ def get_env_port() -> int:
 
 def get_env_host() -> str:
     return os.getenv(ENV_PHOENIX_HOST) or HOST
+
+
+def get_env_trace_endpoint() -> Optional[str]:
+    return os.getenv(ENV_TRACE_ENDPOINT) or None
```

Comment:
```suggestion
    return os.getenv(ENV_TRACE_ENDPOINT)
```

`os.getenv` returns `None` if the environment variable can't be found.